jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/DisplayMessageFragment.java

```diff
@@ -25,6 +31,12 @@
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setRetainInstance(true);
+
+        String no_prev_message = getResources().getString(R.string.no_prev_message);
```

Comment:
Not sure, I have probably been unclear. The answer is: "it doesn't matter, as long as you are consistent". So, we use `camelCase` for variable names, which is the most common in Java/Kotlin/Android. 

There is an exception to the above, where references to UI instances are set for us automatically. This  and instance variables are generated from e.g. `R.id.text_view` for us, and those will exceptionally be in 🐍  case